project1/server/server_tasks/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(server_tasks_lib)
```

Comment:
Ummm, don't you actually have to add at least one `.cpp` file to this library?

Reply:
Should I add all the `.cpp` files from the server_tasks? Do I even need to make this a library?